packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -40,56 +41,193 @@ const LOADPANEL_MESSAGE_CLASS = 'dx-loadpanel-message';
+    QUnit.module('message option tests', () => {
+        ['', undefined, null].forEach(state => {
+            const stateString = (state === '') ? 'empty' : state;
+            QUnit.test(`message is set to ${stateString} on init`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: state,
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                const loadPanelMessage = loadPanel.$wrapper().find('.dx-loadpanel-message').text();
 
-QUnit.module('init', {
-    beforeEach: function() {
-        fx.off = true;
-    },
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
+            });
 
-    afterEach: function() {
-        fx.off = false;
-    }
-}, () => {
-    QUnit.test('rendered markup', function(assert) {
-        const $element = $('#loadPanel').dxLoadPanel({ message: 'Test Loading Message', visible: true }); const $content = $element.dxLoadPanel('instance').$content();
+            QUnit.test(`message is set from ${stateString} to non empty on runtime`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: state,
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                let loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
 
-        assert.ok($element.hasClass(LOADPANEL_CLASS));
-        assert.ok($content.hasClass(LOADPANEL_CONTENT_CLASS), 'Load Indicator created');
-        assert.ok($content.find(MESSAGE_SELECTOR).length);
-        assert.equal($content.find(MESSAGE_SELECTOR).text(), 'Test Loading Message');
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
+
+                assert.timeout(1000);
+                loadPanel.option('message', 'Test');
+                loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
+
+                assert.strictEqual(loadPanelMessage, 'Test', 'Load Panel message value is non-empty and correct value');
+            });
+
+            QUnit.test(`message is set from non empty to ${stateString} on runtime`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: 'Test',
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                let loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
+
+                assert.strictEqual(loadPanelMessage, 'Test', 'Load Panel message value is non-empty and correct value');
+
+                assert.timeout(1000);
+
+                loadPanel.option('message', state);
+                loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
```

Comment:
Can we use the link on the wrapped dom element and call text method in correspond places?

Reply:
[bb57796](https://github.com/DevExpress/DevExtreme/pull/27080/commits/bb57796b0f4e0953f491b0adee68e1f35762a715)